### PR TITLE
plugins/ts-comments: init

### PR DESCRIPTION
**Pull request by khaneliman**
Resolves https://github.com/nix-community/nixvim/issues/2103

### CODE REVIEW

File: plugins/by-name/ts-comments-nvim/default.nix [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          graphql = "# %s";
+          handlebars = "{{! %s }}";
+          hcl = "# %s";
+          html = "";
+          hyprlang = "# %s";
+          ini = "; %s";
+          ipynb = "# %s";
+          javascript = {
+            __default = "// %s";
+            "/* %s */" = null;
```
Comment (traxys) [inline plugins/by-name/ts-comments-nvim/default.nix:42]: Shouldn't this be unkeyed entries?

Comment (traxys) [reply plugins/by-name/ts-comments-nvim/default.nix:42]: Or is it an alternate syntax accepted by the plugin?

Comment (khaneliman) [reply plugins/by-name/ts-comments-nvim/default.nix]: Yeah, just late night stupidity

**Review by MattSturgeon — APPROVED** [review]
I don't see any issues. The pluginDefault _could_ be misleading, if users assume defining one attr will override the others, but that's a minor issue and the solution isn't worth the verbosity.

**Comment by khaneliman** [comment]
@mergifyio queue

**Pull request merged**